kis_api.py

```
@@ -1,372 +1,850 @@
+                "INQR_STRT_DT": today,  # 오늘 날짜
+                "INQR_END_DT": today,   # 오늘 날짜
+                "WCRC_FRCR_DVSN_CD": "02",  # 원화외화구분코드 (02: 외화)
+                "TR_MKET_CD": "00",  # 거래시장코드 (00: 전체)
+                "NATN_CD": "840",  # 국가코드 (840: 미국)
+                "CTX_AREA_FK100": "",
+                "CTX_AREA_NK100": ""
+            }
+
+            self.logger.info(f"[실현손익][조회] 기간: {today} ~ {today}")
+
+            response = requests.get(url, headers=headers, params=params)
```

Ruff (0.14.0)

Probable use of `requests` call without timeout

(S113)